modules/forms/forms-epidemiologia/controller/forms-epidemiologia.controller.ts

```diff
@@ -25,5 +38,96 @@ export async function getLAMPPendientes() {
 
 export async function importLAMPResults() {
     const lamps = await this.getLAMPPendientes();
-    EventCore.emitAsync('notificacion:epidemio:lamp', { lamps } );
+    EventCore.emitAsync('notificacion:epidemio:lamp', { lamps });
+}
+
+async function getRegistrosEjecucionCasoConfirmado(ficha) {
+    let conceptIds = [];
+    const seccion = getSeccionClasificacion(ficha);
+    const segundaclasificacion = seccion?.fields.find(f => f.segundaclasificacion)?.segundaclasificacion;
+
+    if (segundaclasificacion.id === 'confirmado') {
+        conceptIds.push(confirmadoNexoCid);
+    } else {
+        if (segundaclasificacion.id === 'antigeno') {
+            const antigeno = seccion?.fields.find(f => f.antigeno)?.antigeno;
+            if (antigeno.id === 'confirmado') {
+                conceptIds.push(confirmadoTestRapidoCid);
+            } else if (antigeno.id === 'muestra') {
+                conceptIds.push(descartadoTestRapidoCid);
+            }
+        }
+        conceptIds.push(confirmadoPCRCid);
+    }
+
+    return await SnomedCtr.getConcepts(conceptIds);
+}
+
+export function getSeccionClasificacion(ficha) {
+    return ficha.secciones.find(s => s.name === seccionClasificacionName);
+}
+
+export async function crearPrestacionClasificacionFicha(ficha) {
+    const consultaSeguimientoConcept = await SnomedCtr.getConcept(consultaSeguimientoCid, '');
+    const conceptosEjecucion = await getRegistrosEjecucionCasoConfirmado(ficha);
+
+    const registros = conceptosEjecucion.map((c: any) => ({
+        concepto: c,
+        elementoRUP: '594aa21a884431c25d9a0266',
```

Comment:
el id hardcodeado esta mal, esto en demo/test/local no se podría probar.
```suggestion
        const elementosRUPSet = await elementosRUPAsSet();
        const elementoRUPPrestacion = elementosRUPSet.getByConcept(prestacion.solicitud.tipoPrestacion);
```

Así se puede obtener el elementoRUP asociado a un conceptId

Reply:
La variable '**c**' es el concepto.